build.py

```diff
@@ -1329,8 +1328,8 @@ def create_build_dockerfiles(container_build_dir, images, backends, repoagents,
     # since we are using PyTorch, TensorFlow 1, TensorFlow 2 containers that
```

Comment:
Should remove the mention of tensorflow 1 container.

Reply:
Missed that.. removed.